### PR TITLE
0.7.0

### PR DESCRIPTION
**Pull request by paulfri**

### CODE REVIEW

File: default.yml [hunk 1]
```diff
@@ -21,7 +24,7 @@ Layout/CaseIndentation:
 Layout/EndAlignment:
   EnforcedStyleAlignWith: variable
 
-Layout/IndentArray:
+Layout/FirstArrayElementIndentation:
```
Comment (paulfri) [inline default.yml:27]: This was a rename

File: default.yml [hunk 2]
```diff
@@ -71,7 +74,7 @@ Metrics/BlockLength:
 Naming/FileName:
   Exclude: ['Gemfile', 'Guardfile']
 
-Naming/UncommunicativeMethodParamName:
+Naming/MethodParameterName:
```
Comment (paulfri) [inline default.yml:77]: This was a rename

File: default.yml [hunk 3]
```diff
@@ -12,7 +12,10 @@ AllCops:
 Bundler/OrderedGems:
   Enabled: false
 
-Layout/AlignParameters:
+Layout/ArgumentAlignment:
```
Comment (paulfri) [inline default.yml:15]: This is a new cop. Enabled using `with_fixed_indentation` for consistency with `ParameterAlignment` (and also with the api codebase).

`AlignParameters` was renamed to `ParameterAlignment` below

Comment (faun) [reply default.yml:15]: I agree that `with_fixed_indentation` is the correct choice (I'm surprised that this isn't the default).

**Review by faun — APPROVED** [review]
LGTM :cactus:

**Review by timhaines — APPROVED** [review]
LGTM. 👍  Thanks for updating it Paul!

**Pull request merged**